executions/graphql-kotlin-transaction-loader-instrumentation/build.gradle.kts

```diff
@@ -1,12 +1,12 @@
-description = "Transaction Batcher Instrumentation"
+description = "Transaction Loader Instrumentations"
 
 val junitVersion: String by project
 val graphQLJavaVersion: String by project
 val reactorVersion: String by project
 val reactorExtensionsVersion: String by project
 
 dependencies {
-    api(project(path = ":graphql-kotlin-transaction-batcher"))
+    api(project(path = ":graphql-kotlin-server"))
```

Comment:
it seems somewhat odd that instrumentation would depend on the server?

Reply:
oops, forgot to specify it as a `testImplementation` main reason is because for the unit tests im using the `KotlinDataLoader`, we can 

1. Move GraphQL Kotlin DataLoader stuff to a different library
2. Use DataLoaders directly in tests

for now will just  specify `:graphql-kotlin-server`  as a testImplementation